Gravity-Notes/src/Components/TextArea.js/Textarea.css

```diff
@@ -1,6 +1,6 @@
 .textarea-form{
     position: relative;
-    width: 600px;
```

Comment:
use rem everywhere

Reply:
Do not use px as vikram sir also said that above 10px we should always use rem .

Reply:
made changes